src/frontend/handlers.go

```diff
@@ -40,8 +40,9 @@ type platformDetails struct {
 }
 
 var (
-	isCymbalBrand = "true" == strings.ToLower(os.Getenv("CYMBAL_BRANDING"))
-	templates     = template.Must(template.New("").
+	frontendMessage = strings.TrimSpace(os.Getenv("FRONTEND_MESSAGE"))
+	isCymbalBrand   = "true" == strings.ToLower(os.Getenv("CYMBAL_BRANDING"))
+	templates       = template.Must(template.New("").
```

Comment:
The lines starting in:
```
isCymbalBrand ...
template ...
```
didn't change logically. The "change" we're seeing = spacing/formatting changes.